prdoc/pr_9171.prdoc

```diff
@@ -0,0 +1,5 @@
+title: Improve `pr_8860.prdoc`
```

Comment:
I don't think this is needed. It should be possible to add some label to drop pr doc requirement for this pr.

Reply:
I tried to avoid this, but I can't find any label for it. I see that the silent label has been removed.

Reply:
I removed the file, using `R0-no-crate-publish-required` label instead (R0-silent was renamed for clarity)